nxdrive/engine/watcher/local_watcher.py

```diff
@@ -1176,7 +1177,27 @@ def handle_watchdog_event(self, evt: FileSystemEvent) -> None:
             return
         except ThreadInterrupt:
             raise
-        except:
+        except OSError as exc:
+            if exc.errno == 17:
```

Comment:
Does this cover `WindowsError: [Error 183] Cannot create a file when that file already exists`?
Could you also use a named error like `errno.EEXIST`?

Reply:
Yes, `OSError` with `errno == 17` is equivalent to `WindowsError: [Error 183]` 👍